packages/wonder-blocks-tooltip/components/tooltip-anchor.test.js

```diff
@@ -0,0 +1,488 @@
+                        <TooltipAnchor
+                            forceAnchorFocusivity={true}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </View>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                // Act
+                const result = ref && ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBe("0");
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+
+        test("if tabindex already set, leaves it as-is", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <View tabIndex="-1">This is the anchor</View>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+                const nodes = (
+                    <View>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={true}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </View>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                // Act
+                const result = ref && ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBe("-1");
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+    });
+
+    describe("forceAnchorFocusivity is false", () => {
+        test("does not set tabindex on anchor target", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <View>This is the anchor</View>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+                const nodes = (
+                    <View>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={false}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </View>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                // Act
+                const result = ref && ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBeNull();
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+
+        test("if we had added tabindex, removes it", (done) => {
```

Comment:
This test doesn't add `tabindex` anywhere.

Reply:
Yes, it does. It will get added on mount.

Reply:
I've added an extra assert to demonstrate that.